tests/unit/test_parameters/test_parameter_sets_class.py

```diff
@@ -26,26 +25,16 @@ def test_all_registered(self):
         known_entry_points = set(
             ep.name for ep in pybamm.parameter_sets.get_entries("pybamm_parameter_sets")
         )
-        self.assertEqual(set(pybamm.parameter_sets.keys()), known_entry_points)
-        self.assertEqual(len(known_entry_points), len(pybamm.parameter_sets))
+        assert set(pybamm.parameter_sets.keys()) == known_entry_points
+        assert len(known_entry_points) == len(pybamm.parameter_sets)
 
     def test_get_docstring(self):
         """Test that :meth:`pybamm.parameter_sets.get_doctstring` works"""
         docstring = pybamm.parameter_sets.get_docstring("Marquis2019")
-        self.assertRegex(docstring, "Parameters for a Kokam SLPB78205130H cell")
+        assert re.search("Parameters for a Kokam SLPB78205130H cell", docstring)
 
     def test_iter(self):
         """Test that iterating `pybamm.parameter_sets` iterates over keys"""
         for k in pybamm.parameter_sets:
-            self.assertIsInstance(k, str)
-            self.assertIn(k, pybamm.parameter_sets)
-
-
-if __name__ == "__main__":
-    print("Add -v for more debug output")
-    import sys
-
-    if "-v" in sys.argv:
-        debug = True
-    pybamm.settings.debug_mode = True
-    unittest.main()
+            assert isinstance(k, str)
+            assert k in pybamm.parameter_sets
```

Comment:
This test is slightly strange because we iterate `k` over `pybamm.parameter_sets`, so `assert k in pybamm.parameter_sets` will always return `True` over the iterable. This test was redundant, and we can probably remove this assertion. 

```suggestion
            assert isinstance(k, str)
```

Reply:
Agreed.